services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundService.java

```diff
@@ -99,6 +101,8 @@
 
     SwitchInfoData getSwitch(SwitchId switchId);
 
+    SwitchDto setSwitchMaintenance(SwitchId switchId, UnderMaintenanceDto maintenanceDto);
```

Comment:
I'd vote for `SwitchDto setSwitchMaintenance(SwitchId switchId, boolean maintenance, boolean evacuate);`
for better simplicity for caller

Reply:
Fixed.